src/app-layout/visual-refresh/app-bar.tsx

```diff
@@ -57,15 +57,15 @@ export default function AppBar() {
         >
           <InternalButton
             ariaLabel={ariaLabels?.navigationToggle ?? undefined}
-            ariaExpanded={isNavigationOpen}
+            ariaExpanded={isNavigationOpen ? undefined : false}
```

Comment:
Is this basically an "always false" value?

Reply:
Yes, it is false (on hamburger menu button) or unset (close/collapse button).

Reply:
How about removing it, if never activated?

Reply:
It needs _**false**_ value to announce something like "collapsed menu button"

Reply:
I see, I confused the API of `AppLayoutButton` and `InternalButton`. 

Very error-prone to have two components with same property name behaving differently